tools/crm_resource_print.c

```diff
@@ -131,26 +131,21 @@ cli_resource_print(pe_resource_t *rsc, pe_working_set_t *data_set, bool expanded
     return pcmk_rc_ok;
 }
 
-PCMK__OUTPUT_ARGS("attribute-list", "pe_resource_t *", "char *", "GHashTable *")
+PCMK__OUTPUT_ARGS("attribute-list", "pe_resource_t *", "char *", "char *")
```

Comment:
This should be `const char *`. Basically it indicates that this function won't change whatever value is passed in.

(The first `char *` could be const as well, but we don't have to change it now. If you want to, do it in another commit, and update the `va_arg()` line as well, and do the same in `attribute_list_text()`.)